types/redux-actions/index.d.ts

```diff
@@ -85,7 +89,7 @@ export function createAction<Payload, Arg1, Arg2, Arg3, Arg4>(
 
 export function createAction<Payload>(
     actionType: string
-): ActionFunctionAny<Action<Payload>>;
+): ActionFunction1<Payload, Action<Payload>>;
```

Comment:
I find a similar example:
```
  public static create<T0>(): Dispatcher1<T0>;
  public static create<T0, T1>(): Dispatcher2<T0, T1>;
  public static create<T0, T1, T2>(): Dispatcher3<T0, T1, T2>;
```
from https://github.com/Microsoft/TypeScript/issues/1024#issuecomment-68118987 .

And I also test my new definition:

```
const act0 = ReduxActions.createAction('ACTION0')
const act1 = ReduxActions.createAction<string>('ACTION1')

act0().payload // void
act1('hello').payload // string
```

It seem to be expected.